Oop1-Question1 Calculator/BinaryAddition.java

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
for this question we need to approach like this:
Calc{
get-set Button
get set display
get-set Layouts
get-set Operations
}
Button{
get-set Value for digits and operands
we will be might need an Id for it
}
Display{
clear can get text and display result
we will be might need an Id for it
}
Layout{
get and set layouts
we will be might need an Id for it
}
Operations{
all functions and getter setter for function if needed
we will be might need an Id for it
}